lamp_virtual_host

```diff
@@ -64,16 +69,33 @@ check_required_package() {
   fi
 }
 #############################################################
+# Check OS
+#############################################################
+if [ -f /etc/redhat-release ]; then
+  printf "RUN Update\n"
+  sudo yum update
+  
+fi
+
+if [ -f /etc/lsb-release ]; then
+	printf "RUN Update\n"
+	sudo apt-get update && sudo apt-get upgrade
+fi
+#############################################################
 # Check Dependencies
 #############################################################
 # Check Apache
 if [ ! -d /etc/apache2 ]; then
   printf "[${red}${icon_ko}${nocolor}] Sorry, but ${green}/etc/apache2${nocolor} is required.\n"
+  printf "Install Apache2"
+  sudo apt-get install apache2
   exit 1
 fi
 # Check php
 if [ -z "$(ls -A "/etc/php")" ]; then
   printf "[${red}${icon_ko}${nocolor}] Sorry, but ${green}/etc/php${nocolor} is required.\n"
+  printf "Install PHP"
+  sudo apt-get install php7.4
```

Comment:
Stessa cosa qui, con exit 1 si blocca lo script dopo l'installazione. Inoltre qui non basta solo installare php7.4, si dovrebbe trovare un modo per installare sempre l'ultima release (attualmente 8.1) o meglio ancora aggiungere la repo ufficiale di php in modo da averlo sempre aggiornato.